packages/front-end/components/Settings/InviteModal.tsx

```diff
@@ -23,6 +28,28 @@ const InviteModal: FC<{ mutate: () => void; close: () => void }> = ({
   const [emailSent, setEmailSent] = useState<boolean | null>(null);
   const [inviteUrl, setInviteUrl] = useState("");
   const { apiCall } = useAuth();
+  const { data } = useApi<SettingsApiResponse>(`/organization`);
```

Comment:
I just found out that if you try and get this data when logged in as anything but an admin, it's coming back with a 400 error. Is there a different way to get this info?

Reply:
Is that a problem?  Only admins can invite people.

Reply:
It's less of a problem now that I've ripped out the BillingErrorBanner component. The only issue per-say is that this makes the logs a bit messy as it returns a 400. Not sure of a way around that, though.